utils.py

```diff
@@ -72,6 +74,65 @@ def mutations_data(self):
             }
         )
 
+    @staticmethod
+    @numba.njit
+    def calc_mean_tree_arity(num_children_per_tree, num_nodes_per_tree, num_leaves):
+        return num_children_per_tree / (num_nodes_per_tree - num_leaves)
+
+    @staticmethod
+    @numba.njit
```

Comment:
Jitting this won't make any difference because ``np.sum(a)`` is already calling into compiled code. Unless you have loops (or your calling the function from numba jitted code) there's not much point in using numba.